public/css/style.css

```diff
@@ -842,12 +844,33 @@ body * {
     
 }
 
-.source-export,
+.source-export{
+    font-size:0.60em;
+    text-decoration:none;
+    color:#777;
+    margin-left:10px;
+    background: url(images/export.png) no-repeat left center #65a06c;
+    padding-left: 27px;
+    /* left: 0px; */
+    padding-right: 5px;
+    border-radius: 5px;
+    color: #e2dede;
+    height: 30px !important;
+    /* line-height: 30px; */
+    display: inline-block;
+}
 .source-opml {
     font-size:0.60em;
     text-decoration:none;
     color:#777;
     margin-left:10px;
+    background: url(images/import.png) no-repeat left center #65a06c;
```

Comment:
Moreover we are working on rewriting the icons in #1013. It is just next in queue after #1014 and #1054.